contracts/KyberPoolMaster.sol

```diff
@@ -454,12 +784,28 @@ contract KyberPoolMaster is Ownable {
         return delegationFees.length;
     }
 
+    /**
+     * @dev Queries the number of elements in feeHandlersList
+     */
+    function feeHandlersListLength() public view returns (uint256) {
+        return feeHandlersList.length;
+    }
+
     /**
      * @dev Transfers the total amount of a given ERC20 deposited in this contracto a given address
      * @param _token ERC20 token address
      * @param _to address of the receiver
      */
     function claimErc20Tokens(address _token, address _to) external onlyOwner {
+        for (uint256 i = 0; i < feeHandlersList.length; i++) {
+            if (
+                successfulClaimByFeeHandler[feeHandlersList[i]] &&
+                _token == address(rewardTokenByFeeHandle[feeHandlersList[i]])
+            ) {
+                revert("not allowed to claim rewardTokens");
+            }
```

Comment:
I think we can simplify this. The owner shouldn't be able to claim Erc20 tokens of a token associated to a fee handler.
```suggestion
            require(
                _token != address(rewardTokenByFeeHandle[feeHandlersList[i]]),
                "not allowed to claim rewardTokens"
            );
```

By preventing to remove an already claimed feeHandler reward that method makes sure the owner can't workaround this check (by removing the fee Handler, claiming tokens and then adding it again) so all funds are safe